pkgs/development/tools/packcc/default.nix

```diff
@@ -0,0 +1,64 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, bats
+, uncrustify
+, testers
+, packcc
+}:
+
+stdenv.mkDerivation rec {
+  pname = "packcc";
+  version = "1.8.0";
+
+  src = fetchFromGitHub {
+    owner = "arithy";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-T7PWM5IGly6jpGt04dh5meQjrZPUTs8VEFTQEPO5RSw=";
+  };
+
+  dontConfigure = true;
+
+  preBuild = ''
+    cd build/${if stdenv.cc.isGNU then "gcc" else "clang"}
```

Comment:
Please check for clang with `isClang` and add a throw if an unsupported compiler is detected.